webpack.config.js

```diff
@@ -2,18 +2,20 @@ const nodeExternals = require('webpack-node-externals');
 const path = require("path");
 const webpack = require('webpack');
 const { CleanWebpackPlugin } = require("clean-webpack-plugin")
-const CopyWebpackPlugin = require("copy-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin"); 
 
 module.exports = (env, argv) => {
     return {
+        
```

Comment:
@oscarmild Is there any update on the webpack file?

Reply:
@fromgopi Yes we will update the whole file which can build and deploy into the server